### PR TITLE
Polls 20240527

### PR DESCRIPTION
**Pull request by CivicSage3**
version reworked after @blimpa 's comments (thanks)

### CODE REVIEW

**Comment by CivicSage3** [comment]
@VoteWizard @blimpa 
version after blimpa's comments

File: governance/polls/SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-05-27T16:00:00
+end_date: 2024-05-30T16:00:00
+---
+# Poll: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Scope Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, May 27 at 16:00 UTC.
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024.md:20]:
```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, May 27 at 16:00 UTC.
```
I'm removing "Scope" as it is only used in the term Stability Scope Facilitator and not in Governance Facilitators, to keep the document consistent

File: governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes to SparkLend on Gnosis:
+
+- Lowering the security delay for SparkLend governance on Gnosis from 2 days to **0 days**.
```
Comment (VoteWizard) [inline governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md:31]:
```suggestion
- Lower the security delay for SparkLend governance on Gnosis from 2 days to **0 days**.
```

File: governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- Setting the minimum security delay limit to **0**.
```
Comment (VoteWizard) [inline governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md:32]:
```suggestion
- Set the minimum security delay limit to **0**.
```

Comment (VoteWizard) [reply governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md:32]: Good

File: governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+[BA Labs ](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/2) confirm that removing the delay will not impact the risk profile of the Gnosis market.
```
Comment (VoteWizard) [inline governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md.md:34]:
```suggestion
[BA Labs](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/2) confirms that removing the delay will not impact the risk profile of the Gnosis market.
```

**Review by VoteWizard — COMMENTED** [review]
Good job @CivicSage3 and @blimpa. I left a few comments

File: governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+---
+title: SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT and EURe - May 27, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md:2]: Add Oxford comma to match the file name

```suggestion
title: SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024
```

File: governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-05-27T16:00:00
+end_date: 2024-05-30T16:00:00
+---
+# Poll: SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT and EURe - May 27, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md:18]: Add Oxford comma to match the file name

```suggestion
# Poll: SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024
```

File: governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md (hunk 5 above)
Comment (0xldr) [reply governance/polls/SparkLend Ethereum and Gnosis - Turn off Siloed Borrowing for USDC, USDT, and EURe - May 27, 2024.md:2]: or rename the file

File: governance/polls/SparkLend Ethereum and Gnosis - Update Interest Rate Models for ETH, DAI, USDC, USDT, and EURe - May 27, 2024.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+        - Reduce slope 2 from 50% to 15%
+        - Increase optimal utilization from 90% to 95%
+    - USDT
+        - Reduce slope 1 from 12% to 9%
+        - Reduce slope 2 from 50% to 15%
+        - Increase optimal utilization from 90% to 95%
+    - Monerium EUR e-money (EURe)
+        - Reduce slope 1 from 7% to 5%
+        - Reduce slope 2 from 50% to 15%
+        - Increase optimal utilization from 90% to 95%
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum and Gnosis - Update Interest Rate Models for ETH, DAI, USDC, USDT, and EURe - May 27, 2024.md:60]: add deltas where parameters are changing

Comment (0xldr) [reply governance/polls/SparkLend Ethereum and Gnosis - Update Interest Rate Models for ETH, DAI, USDC, USDT, and EURe - May 27, 2024.md:60]: Also consider use of subheadings to separate chains and coins

e.g.

```
## Review

### Ethereum

#### ETH
```

File: governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes to SparkLend on Gnosis:
+
+- Lower the security delay for SparkLend governance on Gnosis from 2 days to **0 days**.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md:31]: clarify that it is the bridge security delay and add the delta

```suggestion
- Lower the bridge security delay for SparkLend governance on Gnosis by 2 days from 2 days to **0 days**.
```

File: governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+[BA Labs](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/2) confirms that removing the delay will not impact the risk profile of the Gnosis market.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+
+- Phoenix Labs will prepare a Spark spell to lower the security delay for Spark governance to 0 days and to set the minimum security delay limit to 0 on Gnosis.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Turn Off Bridge Security Delay  - May 27, 2024.md:42]:
```suggestion
- Phoenix Labs will prepare a Spark spell to lower the bridge security delay for Spark governance to 0 days and to set the minimum security delay limit to 0 on Gnosis.
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Review by 0xldr — APPROVED** [review]

**Pull request merged**